core/services/fluxmonitorv2/orm_test.go

```diff
@@ -185,8 +185,9 @@ func TestORM_CreateEthTransaction(t *testing.T) {
 		payload  = []byte{1, 0, 0}
 		gasLimit = uint32(21000)
 	)
-
+	idempotencyKey := "fluxmonitor-1"
```

Comment:
This might conflict with the other test resulting in locking delays. Can we randomly generate it please?

Reply:
Ah good point. I've updated these tests to use uuid instead now.